.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Follow as per given in issue. Adding reference and comment gives more context. 

```
# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
blank_issues_enabled: false  # default: true
```

Reply:
sure. sorry I missed the part.

Reply:
@anandwana001 Done.